.github/workflows/lint-autoformat.yml

```diff
@@ -21,12 +19,11 @@ jobs:
           fetch-depth: 0
       - name: Run lintrunner (nonretryable)
         continue-on-error: true
-        # we can't run all files here because only changes around where the diff are shown in the PR UI
         run: |
           set -ex
           python3 -m venv /tmp/venv
           source /tmp/venv/bin/activate
-          export ADDITIONAL_LINTRUNNER_ARGS="format"
+          export ADDITIONAL_LINTRUNNER_ARGS="format --all-files"
```

Comment:
Does this automatically run all format rules like ruff and clang format or just PYFMT? We should double check all the lintrunner files that can edit files are marked as formatters

Reply:
For me it runs 
```
  NATIVEFUNCTIONS success!
  CLANGFORMAT success!
  PYFMT success!
  BAZEL_LINTER success!
  NEWLINE success!
  RUFF success!
  MERGE_CONFLICTLESS_CSV success!
  ATEN_CPU_GPU_AGNOSTIC success!
  SET_LINTER success!
ok No lint issues.
```
which are all the ones in .lintrunner.toml that have is_formatter = True